generators/writing/templates/src/_adapters/mongodb.ejs

```diff
@@ -28,5 +28,12 @@
   app.set('mongoClient', promise)<%- sc %>
   <%- insertFragment('more') %>
 }<%- tplJsOnly([sc]) %>
+
+module.exports.registerServicePromise = function (app, promise) {
```

Comment:
@eddyystop Is it possible to do this syntax with the current utils?

Reply:
You have to generate both JS and TS versions for exports/imports.